### PR TITLE
Move some helper files out of models

### PR DESCRIPTION
**Pull request by lunny**

### CODE REVIEW

File: cmd/hook.go [hunk 1]
```diff
@@ -180,12 +180,12 @@ Gitea or set your environment appropriately.`, "")
 	}
 
 	// the environment is set by serv command
-	isWiki := os.Getenv(models.EnvRepoIsWiki) == "true"
-	username := os.Getenv(models.EnvRepoUsername)
-	reponame := os.Getenv(models.EnvRepoName)
-	userID, _ := strconv.ParseInt(os.Getenv(models.EnvPusherID), 10, 64)
-	prID, _ := strconv.ParseInt(os.Getenv(models.EnvPRID), 10, 64)
-	deployKeyID, _ := strconv.ParseInt(os.Getenv(models.EnvDeployKeyID), 10, 64)
+	isWiki := os.Getenv(repo_module.EnvRepoIsWiki) == "true"
```
Comment (delvh) [inline cmd/hook.go]: Is there a reason not to use
```suggestion
	isWiki, _ := strconv.ParseBool(os.Getenv(repo_module.EnvRepoIsWiki))
```

File: cmd/hook.go [hunk 2]
```diff
@@ -314,7 +314,7 @@ func runHookPostReceive(c *cli.Context) error {
 	}
 
 	// Now if we're an internal don't do anything else
-	if os.Getenv(models.EnvIsInternal) == "true" {
+	if os.Getenv(repo_module.EnvIsInternal) == "true" {
```
Comment (delvh) [inline cmd/hook.go]: Again, wouldn't it be better to use
```suggestion
	if isInternal, _ := strconv.ParseBool(os.Getenv(repo_module.EnvIsInternal)); isInternal {
```

File: cmd/hook.go [hunk 3]
```diff
@@ -343,11 +343,11 @@ Gitea or set your environment appropriately.`, "")
 	}
 
 	// the environment is set by serv command
-	repoUser := os.Getenv(models.EnvRepoUsername)
-	isWiki := os.Getenv(models.EnvRepoIsWiki) == "true"
-	repoName := os.Getenv(models.EnvRepoName)
-	pusherID, _ := strconv.ParseInt(os.Getenv(models.EnvPusherID), 10, 64)
-	pusherName := os.Getenv(models.EnvPusherName)
+	repoUser := os.Getenv(repo_module.EnvRepoUsername)
+	isWiki := os.Getenv(repo_module.EnvRepoIsWiki) == "true"
```
Comment (delvh) [inline cmd/hook.go]:
```suggestion
	isWiki, _ := strconv.ParseBool(os.Getenv(repo_module.EnvRepoIsWiki))
```

File: models/repo_list.go [hunk 4]
```diff
@@ -42,6 +42,14 @@ func (repos RepositoryList) Swap(i, j int) {
 	repos[i], repos[j] = repos[j], repos[i]
 }
 
+func valuesRepository(m map[int64]*repo_model.Repository) []*repo_model.Repository {
```
Comment (delvh) [inline models/repo_list.go:46]:
```suggestion
// FIXME: Remove in favor of maps.values when MIN_GO_VERSION >= 1.18
func valuesRepository(m map[int64]*repo_model.Repository) []*repo_model.Repository {
```

**Review by delvh — APPROVED** [review]

**Comment by lunny** [comment]
@delvh All done.

**Review by 6543 — APPROVED** [review]

File: cmd/hook.go [hunk 5]
```diff
@@ -162,7 +162,7 @@ func (n *nilWriter) WriteString(s string) (int, error) {
 }
 
 func runHookPreReceive(c *cli.Context) error {
-	if os.Getenv(models.EnvIsInternal) == "true" {
+	if os.Getenv(repo_module.EnvIsInternal) == "true" {
```
Comment (delvh) [inline cmd/hook.go]:
```suggestion
	if isInternal, _ := strconv.ParseBool(os.Getenv(repo_module.EnvIsInternal)); isInternal {
```

**Review by delvh — APPROVED** [review]

**Comment by 6543** [comment]
.

**Pull request merged**